### ISSUE TITLE
OpenGL extension SGIS_texture_edge_clamp unavailable

### ISSUE THREAD

**Issue by danieljpetersen** [SFML/SFML#1786]
## Subject of the issue

From commit 4fcef893c9d67630ad6d9e9bbf2ee48a62f95eef onwards the following error appears in the terminal, presumably only on nvidia cards

OpenGL extension SGIS_texture_edge_clamp unavailable
Artifacts may occur along texture edges
Ensure that hardware acceleration is enabled if available

## Your environment

* nvidia 3080 GPU, Ubuntu 20.04 (driver 460.39) and Windows 10 (driver 461.33)
* GCC 9.3.0, Visual Studio 2019 

## Steps to reproduce

Any program making use of a texture

```cpp
#include <SFML/Graphics.hpp>

int main()
{
    sf::RenderWindow window(sf::VideoMode(200, 200), "SFML works!");
    sf::CircleShape shape(100.f);
    shape.setFillColor(sf::Color::Green);

    sf::RenderTexture T;
    T.create(200, 200);

   
    while (window.isOpen())
    {
        sf::Event event;
        while (window.pollEvent(event))
        {
            if (event.type == sf::Event::Closed)
                window.close();
        }

        T.clear();
        T.draw(shape);
        T.display();

        sf::Sprite spr;
        spr.setTexture(T.getTexture());

        window.clear();
        window.draw(spr);
        window.display();
    }

    return 0;
}

```

----

I managed to get the error to go away by changing line 11988 in https://github.com/SFML/SFML/blob/master/extlibs/headers/glad/include/glad/gl.h from

```cpp
    SF_GLAD_GL_SGIS_texture_edge_clamp = sf_glad_gl_has_extension(version, exts, num_exts_i, exts_i, "GL_SGIS_texture_edge_clamp");
```
to it's old value

```cpp
    SF_GLAD_GL_SGIS_texture_edge_clamp = sf_glad_gl_has_extension(version, exts, num_exts_i, exts_i, "GL_SGIS_texture_edge_clamp") | sf_glad_gl_has_extension(version, exts, num_exts_i, exts_i, "GL_EXT_texture_edge_clamp");
```

It looks like this an issue in the past which I'm assuming was manually fixed, https://github.com/SFML/SFML/issues/880. I'm assuming that when this file was regenerated the fix didn't get  carried over.

**Comment by eXpl0it3r** [comment]
It's a regression from #1769.

Question is, did we previously manually edit the glad.h file or is there an option for glad to apply this for future generations?

Before
```cpp
    SF_GLAD_GL_SGIS_texture_edge_clamp = sf_glad_gl_has_extension(version, exts, num_exts_i, exts_i, "GL_SGIS_texture_edge_clamp") | sf_glad_gl_has_extension(version, exts, num_exts_i, exts_i, "GL_EXT_texture_edge_clamp");
```

After
```
    SF_GLAD_GL_SGIS_texture_edge_clamp = sf_glad_gl_has_extension(version, exts, num_exts_i, exts_i, "GL_SGIS_texture_edge_clamp");
```

**Comment by binary1248** [comment]
For maintenance reasons, we should handle this in our own code instead of having to remember to manually tweak glad every time we update it (like was done in the past).

**Comment by binary1248** [comment]
This should be fixed in #1789.

**Comment by danieljpetersen** [comment]
Thank you, I just pulled and tested that commit and it's working without error on my rig.
